src/rust/grapl-observe/src/macros.rs

```diff
@@ -0,0 +1,11 @@
+#[macro_export]
+macro_rules! log_time {
+    ($msg:expr, $x:expr) => {{
+        let mut sw = stopwatch::Stopwatch::start_new();
+        #[allow(path_statements)]
+        let result = $x;
+        sw.stop();
+        info!("{} {} milliseconds", $msg, sw.elapsed_ms());
```

Comment:
worth noting we could actually emit a histogram metric on these now, but let's save that for a next diff :)